### ISSUE TITLE
Laravel 5.6 "https" issue when trying to pass form id in blade

### ISSUE THREAD

**Issue by Varin6** [proengsoft/laravel-jsvalidation#309]
I'm trying to make client side validation to work with Laravel 5.6.

My rules in controller:

`protected $rules =
        [
            'first_name' => 'required|min:2|max:32',
            'last_name' => 'required|min:2|max:128',
        ];`

My method:

```
public function show(User $user)
    {
        $validator = JsValidator::make($this->rules);

        return view('users.user', [
            'validator' => $validator,
            'user' => $user,
            'duplicates' => $this->userService->getDuplicates($user)]);
    }
```

My blade view:

`<script type="text/javascript" src="{{ asset('vendor/jsvalidation/js/jsvalidation.js')}}"></script>
    {!! $validator !!}`

The above does not produce any errors but it will not do any validation at all front end. When I try to pass a form ID in the blade file, like this:

`<script type="text/javascript" src="{{ asset('vendor/jsvalidation/js/jsvalidation.js', '#modal-edit-form')}}"></script>
    {!! $validator !!}`

It will produce error: `Uncaught TypeError: $(...).validate is not a function` and it it because it replaces "http://" to "https://" in the code. It only happens when you try to pass the parameter in the blade file. 

I only made it work with this configuration:

Controller method:

```
public function show(User $user)    {
       $validator = JsValidator::make($this->rules, [], [], '#modal-edit-form');

        return view('users.user', [
            'validator' => $validator,
            'user' => $user,
            'duplicates' => $this->userService->getDuplicates($user)]);
    }
```

Blade:

`<script type="text/javascript" src="{{ asset('vendor/jsvalidation/js/jsvalidation.js')}}"></script>
    {!! $validator !!}
`

Seems like the https issue is preventing the browser from finding the .js file on my localhost. Is there a way to change it?

**Comment by Nuranto** [comment]
Second parameter of asset is a bool for secure url....

you can try using 

`    <script type="text/javascript" src="{{ asset('vendor/jsvalidation/js/jsvalidation.min.js')}}"></script>
    {!! $validator->selector('#modal-edit-form') !!}`

**Comment by obahareth** [comment]
Thanks @Nuranto I was facing the same issue then looked at the docs and saw that it was indeed a boolean and was wondering what the heck to do.

**Comment by Nuranto** [comment]
Yes, doc example is wrong here : 
https://github.com/proengsoft/laravel-jsvalidation/wiki/Basic-Usage 
```{{ asset('vendor/jsvalidation/js/jsvalidation.js', '#my-form')}}```
I bet many of us faced this :p

**Comment by obahareth** [comment]
I went ahead and updated these wikis:
* [proengsoft/laravel-jsvalidation/wiki/Basic-Usage](https://github.com/proengsoft/laravel-jsvalidation/wiki/Basic-Usage)
* [proengsoft/laravel-jsvalidation/wiki/Controller-Validation-Example](https://github.com/proengsoft/laravel-jsvalidation/wiki/Controller-Validation-Example)


@Nuranto, can you verify that they're correct now?

**Comment by Nuranto** [comment]
Seems ok! Thanks!

**Comment by bytestream** [comment]
Due to the volume of issues I'm afraid I'm closing this issue due to lack of activity. This avoids fellow collaborators wasting time investigating this issue, and also prevents giving a false impression that it will eventually be addressed.

To the reporter, please reopen if you still encounter this on the latest version providing a minimal test case. You can also contribute directly by submitting a PR. 

Thanks.
